File: js/passwordStrengthTests.js
const isStrongPassword = require("./passwordStrength");

// Valid
console.log(isStrongPassword("Pa$$word1") === true);
console.log(isStrongPassword("ShortA1!") === true);
console.log(isStrongPassword("LongerPassword123!") === true);

// Invalid
console.log(isStrongPassword("short1!") === false);        // too short, no upper
console.log(isStrongPassword("alllowercase") === false);   // no upper, digit, symbol
console.log(isStrongPassword("ALLUPPER1!") === false);     // no lowercase
console.log(isStrongPassword("NoSymbol123") === false);    // no symbol
console.log(isStrongPassword("NoDigit!@#") === false);     // no digit
console.log(isStrongPassword("OnlyLetters!") === false);   // no digit
console.log(isStrongPassword("") === false);

File: python/password_strength_tests.py
from password_strength import is_strong_password

# Valid
print(is_strong_password("Pa$$word1") == True)
print(is_strong_password("ShortA1!") == True)
print(is_strong_password("LongerPassword123!") == True)

# Invalid
print(is_strong_password("short1!") == False)        # too short, no upper
print(is_strong_password("alllowercase") == False)   # no upper, digit, symbol
print(is_strong_password("ALLUPPER1!") == False)     # no lowercase
print(is_strong_password("NoSymbol123") == False)    # no symbol
print(is_strong_password("NoDigit!@#") == False)     # no digit
print(is_strong_password("OnlyLetters!") == False)   # no digit
print(is_strong_password("") == False)
